seq2science/envs/pytxi.yaml

```diff
@@ -0,0 +1,10 @@
+name: pytxi
+channels:
+  - bioconda
+  - conda-forge
+  - defaults
+dependencies:
+  - genomepy=0.11.1
+  - pip
+  - pip:
+    - git+https://github.com/vanheeringen-lab/pytxi.git@develop
```

Comment:
Not sure if this is a permanent solution, but I think we should at least use a tagged version or sth like that..

Reply:
yea sure! I'm running into some issue during my test run. Once those are fixed it use a commit hash/version tag!